cmd/github-assets-uploader/main.go

```diff
@@ -115,3 +127,21 @@ func uploadAsset(repoOwner, repoName, tag, assetPath, mediaType, token string, o
 	glog.Infof("Upload asset succeed, id %d, name '%s', url: '%s'\n", releaseAsset.GetID(), releaseAsset.GetName(), releaseAsset.GetBrowserDownloadURL())
 	return nil
 }
+
+func getDraftRelease(ctx context.Context, client *github.Client, repoOwner, repoName string) (*github.RepositoryRelease, error) {
```

Comment:
One last thing: would you please rename this function to `getReleaseByName`, and put the `releaseName` as parameter? Make sure only `main` accesses the `flags` var.

Reply:
sure, Done.